src/provider_info_collector.js

```
@@ -0,0 +1,884 @@
+          }
+        }
+        break;
+
+      default:
+        // Generic extraction for other providers
+        const genericPatterns = [
+          new RegExp(`${provider}[^"\\s<>]*[0-9][^"\\s<>]*`, 'gi'),
+          /[a-zA-Z]+-[0-9]+[bBmM]?-?[a-zA-Z]*[0-9]*[kKmMbB]?/g
+        ];
+        
+        genericPatterns.forEach(pattern => {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)